R/package.R

```diff
@@ -97,47 +98,54 @@ re_match <- function(pattern, text, ...) {
 
 re_match_all <- function(pattern, text, ...) {
 
-  stopifnot(is.character(pattern), length(pattern) == 1, !is.na(pattern))
+  stopifnot(is.character(pattern), length(pattern) == 1, !is.na(pattern), length(text) > 0)
```

Comment:
I am not sure which way is better,  so you get the right number of columns with zero length text? If it is always a 1x0 df that seems worse than an error to me. 

If the dimension is stable then I agree.